samples/nrf5340/multiprotocol_rpmsg/prj.conf

```diff
@@ -0,0 +1,21 @@
+CONFIG_LOG=y
+
+CONFIG_RPMSG_SERVICE=y
+CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
```

Comment:
I think it's enabled indirectly by `NRF_802154_SER_RADIO` and BT HCI RPMSG options. Isn't it?

Reply:
Looks like that's the case. Removed.